debian/cuttlefish-common.init

```diff
@@ -197,13 +222,18 @@ destroy_bridged_interfaces() {
 
 start() {
     # Enable ip forwarding
-    echo 1 >/proc/sys/net/ipv4/ip_forward
+    echo 1 > /proc/sys/net/ipv4/ip_forward
+    echo 1 > /proc/sys/net/ipv6/conf/all/forwarding
```

Comment:
If you aren't NAT'ing any more, do you need ipv6 forwarding?

Reply:
I tried turning off ipv6 forwarding. The device got IPv6 addresses and dropped them in a few seconds. Eventually it had an IPv4 address but no IPv6 ones.
When ipv6 forwarding is on, the device got persistent IPv6 addresses even though it could not access the Internet. I have no idea why ipv6 forwarding made the difference.